config/initializers/message_bus.rb

```diff
@@ -1,5 +1,14 @@
 # frozen_string_literal: true
 
+# Monkey patching message_bus to enforce redis namespace
+require 'message_bus/backends/redis'
+
+class MessageBus::Redis::ReliablePubSub
+  def new_redis_connection
+    Redis::Namespace.new('system-mbus', redis: ::Redis.new(@redis_config))
```

Comment:
We should be able to configure the namespace it in case it is conflicting as an enhancement, but I think good enough for now

Reply:
I'll do that too. Not a huge amount of effort anyway.